app/views/order_delivery_mailer/delivery_email.html.erb

```diff
@@ -4,8 +4,12 @@
     <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
   </head>
   <body>
-    <h1>Welcome <%= @order.employee.email %>!</h1>
-    <p> Your order "<%= @order.reward.title %>" has been delivered.</p>
+    <% if @order.employee.first_name.nil? %>
```

Comment:
this should go to employee model - so you simply call a method like` employee.display_name` or somethig like this, and do not really care, if the employee has a name, email or not :)